uint8 request
uint8 REQUEST_START = 53 #0x35
uint8 REQUEST_ABORT = 48 #0x30
---
int8 result
int8 RESULT_FAIL = -1
int8 RESULT_DONE = 0